e2e/integration/menu.e2e.ts

```diff
@@ -4,7 +4,13 @@ describe('Menu', () => {
   });
 
   it('should have valid items count', () => {
-    cy.get('.menu-content').find('li').should('have.length', 34);
+    cy.get('.menu-content').find('li').should('have.length', 35);
+  });
+
+  it('should have a back button', () => {
+    cy.contains('a', 'Back to Docs')
+      .should('have.attr', 'href')
+      .and('contain', 'https://mongodb.com/docs/');
```

Comment:
I originally wanted this to be split into a couple of unit tests that would test for both default and custom behaviors, but I kept running into the following import issue:

```
Test suite failed to run

    TypeError: common_elements_1.headerCommonMixin is not a function

      56 |
      57 |   h1 {
    > 58 |     ${headerCommonMixin(1)};
         |       ^
      59 |     color: ${props => props.theme.colors.primary.main};
      60 |     margin-top: 0;
      61 |   }

      at Object.<anonymous> (src/components/Markdown/styled.elements.tsx:58:7)
      at Object.<anonymous> (src/components/Markdown/SanitizedMdBlock.tsx:6:1)
      at Object.<anonymous> (src/components/Markdown/Markdown.tsx:4:1)
      at Object.<anonymous> (src/components/PayloadSamples/MediaTypeSamples.tsx:7:1)
      at Object.<anonymous> (src/components/SchemaDefinition/SchemaDefinition.tsx:8:1)
      at Object.<anonymous> (src/services/AppStore.ts:14:1)
      at Object.<anonymous> (src/services/index.ts:1:1)
      at Object.<anonymous> (src/components/Redoc/Redoc.tsx:8:1)
      at Object.<anonymous> (src/components/RedocStandalone.tsx:10:1)
      at Object.<anonymous> (src/components/index.ts:1:1)
      at Object.<anonymous> (src/index.ts:1:1)
      at Object.<anonymous> (src/services/models/Webhook.ts:5:1)
      at Object.<anonymous> (src/services/SpecStore.ts:5:1)
      at Object.<anonymous> (src/services/models/index.ts:1:1)
      at Object.<anonymous> (src/services/MenuBuilder.ts:4:1)
      at Object.<anonymous> (src/services/MenuStore.ts:6:1)
      at Object.<anonymous> (src/common-elements/panels.ts:1:1)
      at Object.<anonymous> (src/common-elements/index.ts:1:1)
      at Object.<anonymous> (src/components/SideMenu/styled.elements.ts:5:1)
      at Object.<anonymous> (src/components/SideMenu/SideMenuBackButton.tsx:2:1)
```
Attempting to import a different side menu subcomponent, `MenuItem`, was resulting in the same error. Not sure why this is happening, but this might be okay/expected considering the minimal amount of component unit tests present 🤔

Reply:
Hmm, I think this is fine too? But it is odd. Let me look at this more tomorrow!! But for me, this is not a blocker for merging.